makederiv

```diff
@@ -94,14 +96,23 @@ elif [[ "${OUTPUT_TYPE}" == "dvd" ]] ; then
     EXTENSION="iso"
     RELATIVEPATH="objects/access/dvd"
     DEPENDENCIES+=(dvdauthor mkisofs)
+elif [[ "${OUTPUT_TYPE}" == "m4a" ]] ; then
+    EXTENSION="m4a"
+    RELATIVEPATH="objects/access/m4a"
```

Comment:
can remove and let the podcast section handle this.

Reply:
seems like the other m4a sections below could go as well, since they are now redundant to the podcast option.

Reply:
removed here and below